src/instructlab/model/backends/vllm.py

```diff
@@ -259,6 +283,13 @@ def build_vllm_cmd(
             verify_template_exists(path)
             vllm_cmd.extend(["--chat-template", chat_template])
 
+    # Auto-detect whether model is quantized w/ bitsandbytes and add potentially missing args
+    if is_bnb_quantized(model_path):
+        if not contains_argument("--quantization", vllm_args):
+            vllm_cmd.extend(["--quantization", "bitsandbytes"])
+        if not contains_argument("--load-format", vllm_args):
+            vllm_cmd.extend(["--load-format", "bitsandbytes"])
+
```

Comment:
I think these should be called before `is_bnb_quantized` so that we don't call `is_bnb_quantized` (and avoid loading the json etc) if the arguments already contain `"--quantization"` and `"--load-format"`.